schemas/v3.1/schema.yaml

```
@@ -0,0 +1,809 @@
+      OAuthFlows:
+        $anchor: OAuthFlows
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          implicit:
+            allOf:
+              - required:
+                  - scopes
+              - $ref: '#CommonFlow'
+              - $ref: '#AuthorizationFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+          password:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+          clientCredentials:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+          authorizationCode:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#AuthorizationFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+        unevaluatedProperties: false
+        $defs:
+          Common:
+            $anchor: CommonFlow
+            properties:
+              refreshUrl:
+                type: string
+                format: uri-reference
+              scopes:
+                type: object
+                additionalProperties:
+                  type: string
+          Token:
+            $anchor: TokenFlow
+            required:
+              - tokenUrl
+            properties:
+              tokenUrl:
+                type: string
+                format: uri-reference
+          Authorization:
+            $anchor: AuthorizationFlow
+            required:
+              - authorizationUrl
+            properties:
+              authorizationUrl:
+                type: string
+                format: uri-reference
+      Link:
+        $anchor: Link
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            oneOf:
+              - properties:
+                  operationRef:
+                    type: string
+                    format: uri-reference
+                  operationId: false
+              - properties:
+                  operationRef: false
+                  operationId:
+                    type: string
+            properties:
+              parameters:
+                type: object
+                additionalProperties: true
+              requestBody: true
+              server:
+                $ref: '#Server'
+            unevaluatedProperties: false
+      Callback:
+        $anchor: Callback
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
```

The `Callback` schema is missing the actual contents of callbacks i.e. path items. See the [3.0 schema](https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v3.0/schema.yaml#L975-L980) for the reference.
```suggestion
            allOf:
              - $ref: '#/$defs/Mixins/$defs/Extensible'
            additionalProperties:
              $ref: '#PathItem'
```